app/models/jira_issue.rb

```diff
@@ -0,0 +1,42 @@
+# == Schema Information
+#
+# Table name: jira_issues
+#
+#  id              :bigint           not null, primary key
+#  environment     :enum
+#  informed_at     :datetime         not null
+#  issue_type      :enum             not null
+#  key             :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  jira_project_id :bigint           not null
+#
+# Indexes
+#
+#  index_jira_issues_on_jira_project_id  (jira_project_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (jira_project_id => jira_projects.id)
+#
+
+class JiraIssue < ApplicationRecord
+  enum issue_types: { bug: 'bug', task: 'task', epic: 'epic', story: 'story' }
```

Comment:
can't you use symbols here? like `{ bug: :bug, task: :task, epic: :epic, story: :story }`